untitled/main.cpp

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+#include <iomanip>
+
+int main() {
+    //Stock Transactions: Michael Watkins 1-31-16
+
+    //Declarations and Initialization
+    double SHARES_BOUGHT = 1000;
+    double SHARES_SOLD = 1000;
+    double COMMISION_RATE = 0.02;
+    double PURCHASE_PRICE = 45.50;
+    double SALE_PRICE = 56.90;
+
+    //Sale Calculations
+    double STOCK_BOUGHT = SHARES_BOUGHT * PURCHASE_PRICE;
```

Comment:
Consider a more descriptive name for this variable. Something like `stock_purchase_price`